executable-spec/src/Cardano/Ledger/Generators/QuickCheck.hs

```diff
@@ -80,17 +82,31 @@ boundedWith aMinBound aMaxBound maxDifference =
   where mid = aMinBound + (aMaxBound - aMinBound) `div` 2
 
 prvNoQuorum :: Gen Word8
-prvNoQuorum = Gen.choose (3, 7)
+prvNoQuorum =  Gen.choose (1, 3)
 
 prvNoMajority :: Gen Word8
-prvNoMajority = Gen.choose (3, 7)
+prvNoMajority = Gen.choose (1, 3)
 
 rA :: Gen Float
 rA = Gen.choose (0, 0.5)
 
-stakeDist :: Gen (StakeDistribution p)
-stakeDist = do
-  someParticipants <- participants
+stakeDist :: (Participants p) -> Gen (StakeDistribution p)
+stakeDist someParticipants = do
+  Gen.oneof [ stakeDistUniform someParticipants
+            , stakeDistSkewed someParticipants
+            ]
+
+stakeDistUniform :: (Participants p) -> Gen (StakeDistribution p)
+stakeDistUniform someParticipants = do
   let vkeys = Set.toList $ dom someParticipants
   stks <- Gen.vectorOf (length vkeys) (Gen.choose (1, 20))
   pure $ StakeDistribution $ Map.fromList $ zip vkeys (Stake <$> stks)
+
+stakeDistSkewed :: (Participants p) -> Gen (StakeDistribution p)
+stakeDistSkewed someParticipants = do
+  let vkeys = Set.toList $ dom someParticipants
+  stksBig <- Gen.vectorOf (round @Float $ (fromIntegral $ length vkeys) * 0.20) (Gen.choose (1000, 2000))
```

Comment:
It seems this `0.20` is implicitly coupled to the coverage tests. We should at least mention this in the comments. 

The range `(1000, 2000)` is quite large, we should probably reduce this. The range of the "small" stakeholders is `(1, 20)` so we might want to make this `(1000, 1000 + 20)`.

Reply:
Done OK.